sass/_theme_layout.sass

```diff
@@ -2,6 +2,15 @@
 html
   font-size: $base-font-size
 
+// Set uniform form font styles
+// This prevents Firefox on Linux from rendering forms with dark styling
+// when using a dark system theme, which looks bad on an otherwise light
+// website theme
+input
+  -moz-appearance: none
```

Comment:
Is just the bg and text color not enough to change the appearance?

Reply:
I can't see any practical difference with `-moz-apperance` none, presumably because there's some custom styling on the input form already (such as rounded corners). If you want, I can remove it.

Reply:
If it has no effect, it's better to remove it